### ISSUE TITLE
Friendly note on missing accessibility features

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#8]
Hi there! I really appreciate the work put into this project. I just wanted to point out that accessibility features seem to be missing at the moment. It would be great to see improvements in this area to make the project more inclusive for everyone. Thanks for considering!
